patches/tModLoader/Terraria/release_extras/start-tModLoader.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-cd "$(dirname "$0")"
```

Comment:
The primary concern with directly exit-ing is that it does not tell the user why it failed nor us that the issue was with the CD execution.

As an alternative, I'd generally prefer instead of directly exitting that we write to the terminal and then request input from user to acknowledge that it failed (or give a chance for a screenshot). 

Especially important since Mac users are also involved with this same script and not confident their terminal would stay visible on exit